mirrormaker_replication_flow.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2018 Aiven, Helsinki, Finland. https://aiven.io/
+
+package aiven
+
+type (
+	// MirrorMakerReplicationFlowHandler is the client which interacts with the
+	// Kafka MirrorMaker 2 ReplicationFlows endpoints on Aiven.
+	MirrorMakerReplicationFlowHandler struct {
+		client *Client
+	}
+
+	// MirrorMakerReplicationFlowRequest request used to create a Kafka MirrorMaker 2
+	// ReplicationFlows entry.
+	MirrorMakerReplicationFlowRequest struct {
+		Enabled         bool     `json:"enabled"`
+		SourceCluster   string   `json:"source_cluster"`
+		TargetCluster   string   `json:"target_cluster"`
+		Topics          []string `json:"topics"`
+		TopicsBlacklist []string `json:"topics.blacklist"`
+	}
+
+	// MirrorMakerReplicationFlowResponse represents the response from Aiven after
+	// interacting with the Kafka MirrorMaker 2 API.
+	MirrorMakerReplicationFlowResponse struct {
+		APIResponse
+		ReplicationFlows MirrorMakerReplicationFlowRequest `json:"replication_flows"`
```

Comment:
Would seem cleaner if there was separate struct for representing replication flow instead of reusing the request struct in responses.